dist/index.html

```diff
@@ -0,0 +1,97 @@
+<head><link href="dist/../dist.css" rel="stylesheet"></head>Html Webpack Plugin:
```

Comment:
Missing a file here? This is a failed compilation since it cannot get `index.ejs` (So not working atm)

Reply:
I think wbepack has been setup to put this dist js/css/html inside the root... and this index.html just didnt get updated from before that change. Is this supposed the be the case or shall we change it?

Reply:
If we have the index in the root, we can just whack this one. Strange that it will be show though. (Either way, it is broken, so cannot work - best is to whack it)